packages/php-table-backend-utils/src/Table/Bigquery/BigqueryTableReflection.php

```diff
@@ -190,13 +190,19 @@ public function getPartitionsList(): array
         $partitions = [];
         /**
          * @var array{
-         *      partition_id: string,
+         *      partition_id: string|null,
          *      total_rows: int,
          *      last_modified_time: Timestamp,
          *      storage_tier: string
          *  } $partition
          */
         foreach ($result as $partition) {
+            if ($partition['partition_id'] === null) {
+                // by default table has one unnamed partition
+                // this would be confusing as partitioning is not set in this case
+                // ignore this partition without id
+                continue;
+            }
```

Comment:
tady sem to okomentoval tabulka má partition ikdyž nemá partitioning, dává to smysl, ale nedává to smysl vracet.